content/04.results.md

```diff
@@ -123,12 +123,13 @@ We show the publication rate of preprints since bioRxiv first started.
 The x-axis represents months since bioRxiv started, and the y-axis represents the proportion of preprints published given the month they were posted.
 The light blue line represents the publication rate previously estimated by Abdill et al. [@doi:10.7554/eLife.45133].
 The dark blue line represents the updated publication rate using only CrossRef-derived annotations, while the dark green line includes annotations derived from our embedding space approach.
-The horizontal lines represent the overall proportion of preprints published as of the time of the annotation snapshot.
+The horizontal lines represent the overall proportion of preprints published since the time of accessing the annotated snapshot.
```

Comment:
I don't understand how this is the proportion of preprints published since when you accessed the annotated snapshot. Isn't it the proportion published now?

Reply:
>I don't understand how this is the proportion of preprints published since when you accessed the annotated snapshot. Isn't it the proportion published now?

My wording is off. Yes, it is the proportion published to date (excluding preprints not accounted for in the downloaded snapshot). I was trying to update this sentence based on what the reviewer asked (reproduced below), but realize it's confusing now. I'll update to clear confusion.

>adjusting for recency? i.e. not sampling 2019 preprints? in figure C the line indicates (if I'm interpreting correctly) that overall only 46.55% are published but that's because it includes very recent preprints that haven't had time to be journal published yet. Just be explicit that you are adjusting for recency (i.e. excluding 2019 and newer preprints) when you say that most preprints are eventually published.

Reply:
It looks to me like you got this - is that right? If so 👍